### ISSUE TITLE
Pelvis Hypermobility

### ISSUE THREAD

**Issue by samardeep331** [obophenotype/human-phenotype-ontology#11209]
**Preferred term label:**
Pelvis Hypermobility

**Synonyms**
Hands on Floor (Spine) Hypermobility

**Definition (free text, please give PubMed ID)**
The ability to bend forward and place the palms of one's hands flat on the floor in front of one's feet without bending one's knees.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ehlers-Danlos Syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@samardeep331 
I am not sure this finding always indicates pelvis hypermobility.
I would propose we label this
Positive trunk flexion test
syn:
Trunk flexion with palms on floor
Positive palms-to-floor maneuver
Increased forward flexion of the trunk
Abnormal forward flexion of the trunk manoeuvre
definition: The proband is asked to flex the trunk forward with knees straight. The result of the test is considered positive (abnormal) if the palms can be made to rest flat on the floor.


Comment: This item is elicited as one of the nine points of the Beighton Score (BS), a set of manoeuvres in a nine-point scoring system, used as the standard method of assessment for Generalised Joint Hypermobility.
PMID: [33738549](https://pubmed.ncbi.nlm.nih.gov/33738549/)

Thoughts?

**Comment by pnrobinson** [comment]
adding as
Adding new term: 
- ID: HP:6001330
- label: Positive trunk flexion test
- parents: [Joint hypermobility (HP:0001382)](https://hpo.jax.org/app/browse/term/HP:0001382)
- pmid: PMID:33738549
- def: The proband is asked to flex the trunk forward with knees straight. The result of the test is considered positive (abnormal) if the palms can be made to rest flat on the floor.
- comment: This item is elicited as one of the nine points of the Beighton Score (BS), a set of manoeuvres in a nine-point scoring system, used as the standard method of assessment for Generalised Joint Hypermobility.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11209
- synonyms: Increased forward flexion of the trunk(NONE)|Positive palms-to-floor maneuver(NONE)|Trunk flexion with palms on floor(NONE)|Abnormal forward flexion of the trunk manoeuvre(NONE)
